corehq/apps/accounting/models.py

```diff
@@ -782,7 +782,12 @@ def user_facing_description(self):
         from corehq.apps.accounting.user_text import DESC_BY_EDITION, FEATURE_TYPE_TO_NAME
 
         def _default_description(plan, monthly_limit):
-            if plan.edition != SoftwarePlanEdition.ENTERPRISE:
+            if plan.edition in [
+                SoftwarePlanEdition.COMMUNITY,
+                SoftwarePlanEdition.STANDARD,
+                SoftwarePlanEdition.PRO,
+                SoftwarePlanEdition.ADVANCED,
+            ]:
```

Comment:
nit: instead of maintaining a list here, can this be added in `SoftwarePlanEdition` as plans that need to have monthly limit (name suggestion: plans_with_montly_limit) defined so that in case we add a new one with monthly limit in `SoftwarePlanEdition`, this would not be missed.

Reply:
I think I'll leave as-is for now since this list hasn't changed in years, but will keep that in mind in case anything changes.